holoviews/plotting/bokeh/util.py

```diff
@@ -587,7 +587,7 @@ def get_tab_title(key, frame, overlay):
             title = []
             if frame.label:
                 title.append(frame.label)
-                if frame.group != frame.params('group').default:
+                if frame.group != frame.param['group'].default:
```

Comment:
Annoying I know but this avoids making an instance parameter when it's not needed.

```suggestion
                if frame.group != frame.param.objects('existing)['group'].default:
```